### ISSUE TITLE
Personal notes broken again again

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#5455]
### Actual behavior
Personal notes not available. Page works but no notes show up.

### Steps to reproduce the problem
Go to .../notes. No notes available.

### Friendica version you encountered the problem
2018.08-dev

### Friendica source (git, zip)
git

### PHP version
7.0

### SQL version
10.0 MariaDB

**Comment by tobiasd** [comment]
I am on a27ef8d0c9c835724ad7682c920f396420c3681b and personal notes are working here fine. Which commit are you on? (`git log` will tell you in the first line)

**Comment by AlfredSK** [comment]
Loading of the .../notes page takes a lot of time. No errors in FF web console.
Database: 0/0, Network: 0, Rendering: 0, Session: 0, I/O: 0, Other: **74.94**, Total: 74.94

git log: commit ea24ac9d950125d83e3b6a8f2abcaa24a0f316f9

Grrr. Logfile is empty because of the changes to the log levels.

**Comment by AlfredSK** [comment]
![bildschirmfoto von 2018-07-23 16-31-41](https://user-images.githubusercontent.com/3656054/43083102-012b25a4-8e96-11e8-83ef-fa9ae8d4fbf4.png)

I switched to vier for testing but I don't get any notes with vier as well.

**Comment by AlfredSK** [comment]
When I create a new note (tried it on mordor.social) the note vanishes right after saving it.

**Comment by AlfredSK** [comment]
Pulled again. Now on commit 2d0a749734bb33be00b9674cb34582c602ff0d74
Notes still don't work.

**Comment by tobiasd** [comment]
Is there anything in the logs?

**Comment by annando** [comment]
I will check after I updated my system to the latest version.

**Comment by AlfredSK** [comment]
No, there isn't anything in the logs (loglevel INFO). Also there is no PHP error or warning. Firefox web console is showing no errors (HTTP 1.1 200 OK).
Looks like the query result is simply empty.

If I knew the query I could check that in mysql shell.

**Comment by AlfredSK** [comment]
Is it this query?

````
SELECT `item`.`id`, `item`.`id` AS `internal-iid`, `item`.`network` AS `internal-network`, `item`.`icid` AS `internal-icid`, `item`.`iaid` AS `internal-iaid` FROM `item` STRAIGHT_JOIN `contact` ON `contact`.`id` = `item`.`contact-id` AND NOT `contact`.`blocked` AND ((NOT `contact`.`readonly` AND NOT `contact`.`pending` AND (`contact`.`rel` IN (2, 3))) OR `contact`.`self` OR `item`.`gravity` != 0 OR `contact`.`uid` = 0) STRAIGHT_JOIN `contact` AS `author` ON `author`.`id` = `item`.`author-id` AND NOT `author`.`blocked` STRAIGHT_JOIN `contact` AS `owner` ON `owner`.`id` = `item`.`owner-id` AND NOT `owner`.`blocked` LEFT JOIN `user-item` ON `user-item`.`iid` = `item`.`id` AND `user-item`.`uid` = 2 WHERE (`item`.`uid` = 2 AND `item`.`post-type` = 128 AND `item`.`gravity` = 0 AND `item`.`wall` = 0 AND `item`.`allow_cid` = '<2>' AND `item`.`contact-id` = 2) AND `item`.`visible` AND NOT `item`.`deleted` AND NOT `item`.`moderated` AND (`user-item`.`hidden` IS NULL OR NOT `user-item`.`hidden`) ORDER BY `item`.`created` DESC LIMIT 0, 40;
````

**Comment by tobiasd** [comment]
The bug finally hit my notes...

**Comment by tobiasd** [comment]
some 20 items from my uid.

**Comment by annando** [comment]
The first or the second query? Or both?

**Comment by tobiasd** [comment]
The 2nd returns an empty set.

**Comment by AlfredSK** [comment]
@annando Do you need anything from me? I see a PR. :-)

**Comment by annando** [comment]
Please execute this query:
```SELECT `id`, `uid`, `type`, `psid`, `deleted` FROM `item` WHERE `post-type` = 128;```
I'm crossing the fingers that only ```psid``` is ```null```, but that the record isn't ```deleted```.

**Comment by AlfredSK** [comment]
Oh, oh! I saved a lot of admin stuff as notes...

**Comment by AlfredSK** [comment]
`SELECT `id`, `uid`, `type`, `psid`, `deleted` FROM `item` WHERE `post-type` = 128;`

70 rows in set (43.22 sec)

`SELECT `id`, `uid`, `post-type` FROM `item` WHERE `type` = 'note';`

31 rows in set (4.36 sec)

**Comment by annando** [comment]
Question is, if "deleted" is set or not. If not then you don't have a problem. if yes, then I hope that you have got some backup. I'm really sorry for that.

**Comment by AlfredSK** [comment]
Hmm... I changed the query to show only the records for my UID.

````
MariaDB [friendica]> SELECT `id`, `uid`, `type`, `psid`, `deleted` FROM `item` WHERE `post-type` = 128 AND `uid` = 2;
+----------+-----+------+------+---------+
| id       | uid | type | psid | deleted |
+----------+-----+------+------+---------+
|   589797 |   2 | NULL |   16 |       1 |
|  3330496 |   2 | NULL |   16 |       1 |
|  3434508 |   2 | NULL |   16 |       1 |
|  3811472 |   2 | NULL |   16 |       0 |
|  2307740 |   2 | NULL |   16 |       1 |
|  5173324 |   2 | NULL |   16 |       0 |
|  5457507 |   2 | NULL |   16 |       0 |
|  5767391 |   2 | NULL |   16 |       0 |
| 10114602 |   2 | NULL |  338 |       0 |
| 10138946 |   2 | NULL |  338 |       0 |
+----------+-----+------+------+---------+
10 rows in set (2.04 sec)
````

**Comment by Quix0r** [comment]
I have created a test note:
````
id      | uid | type | psid | deleted
------- | --- | ---- | ---- | --
1539321 | 1   | NULL | 4    | 0
````

**Comment by Quix0r** [comment]
Idea for performance reasons: Can we get this data out into a separate table? Selecting on `items` table takes a lot of time.

**Comment by Quix0r** [comment]
That maybe will solve it only partly as then people have to increase key cache on an already overloaded table. Let's split this data here, if possible with own model class.

**Comment by annando** [comment]
Notes are nothing more than items. So it doesn't make sense to duplicate the coding there. I see other priorities.

**Comment by AlfredSK** [comment]
Well, bad news. :-) The old notes are showing up again. But if I create _a new note_ it is only visible on the profile/status page but _not on the notes page_. :-/

Can someone reopen this issue or shall I create a new one?

**Comment by annando** [comment]
I will have a look at it - but not today, I guess.

**Comment by tobiasd** [comment]
> Well, bad news. :-) The old notes are showing up again. But if I
> create _a new note_ it is only visible on the profile/status page but
> _not on the notes page_. :-/

confirmed

**Comment by tobiasd** [comment]
So, they are now shown at `/network` and `/notes` but not on the profile page when logged in. Is that the expected way?

**Comment by annando** [comment]
They are shown on /network? You aren't kidding me at this early morning?

Args ...

**Comment by tobiasd** [comment]
They ere always shown there IIRC just as your normal postings was wondering about your wall.

**Comment by annando** [comment]
So it isn't a bug that they are shown there? The "wall" field is controlling only if an item is displayed on the profile page.
